lib/check.mli

```diff
@@ -0,0 +1,33 @@
+val dune_checks :
+  dry_run:bool ->
+  skip_build:bool ->
+  skip_tests:bool ->
+  pkg_names:string list ->
+  log_status:bool ->
+  Fpath.t ->
+  (int, [ `Msg of string ]) result
+
+(* checks if the packages in [pkg_names] - or, if [pkg_names] is empty, all packages in [dir] - build and pass their tests; depending on [log_status] with or without status logs. 
+  if [skip_build] or [skip_tests], the correspondent checks are skipped. *)
```

Comment:
Why is this not a documentation comment?

Reply:
Good catch!